Modules/Devices/src/Devices.Application/Identities/Commands/UpdateDeletionProcesses/Handler.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using MediatR;
+using Microsoft.Extensions.Logging;
+
+namespace Backbone.Modules.Devices.Application.Identities.Commands.UpdateDeletionProcesses;
+public class Handler(IIdentitiesRepository identitiesRepository, ILogger<Handler> logger) : IRequestHandler<UpdateDeletionProcessesCommand, UpdateDeletionProcessesResponse>
+{
+    public async Task<UpdateDeletionProcessesResponse> Handle(UpdateDeletionProcessesCommand request, CancellationToken cancellationToken)
+    {
+        var response = new UpdateDeletionProcessesResponse
```

Comment:
Why this type of declaration and not something like `var response = new UpdateDeletionProcessesResponse();`